src/components/HomePage/SearchResultsArea/Offer/OfferItem.js

```diff
@@ -67,7 +67,7 @@ const OfferItem = ({ offer, offerIdx, selectedOfferIdx, setSelectedOfferIdx, loa
                 <ListItemAvatar>
                     {loading ?
                         <Avatar data-testid="avatar-skeleton" variant="circular" classes={{ colorDefault: "transparent" }}>
-                            <Skeleton circle width={100} height={100} />
+                            <Skeleton circle="true" width={100} height={100} />
```

Comment:
I made this change because of this new warning that appeared:

![image](https://user-images.githubusercontent.com/50210213/160244301-4bbc23a1-aee8-4dd5-80ac-607e76bd58dc.png)